docs/src/main/sphinx/release/release-396.md

```diff
@@ -0,0 +1,51 @@
+# Release 396 (14 Sep 2022)
+
+## General
+
+* Improve performance of queries that process string data types. ({issue}`12798`)
+* Fix failure when querying a view which uses a table function.  ({issue}`13944`)
+
+## BigQuery connector
+
+* Add support for [column comments and table comments](/sql/comment). ({issue}`13882`)
+
+## ClickHouse connector
+
+* Allow pushdown of larger predicates by default. ({issue}`14029`)
+* Extend the range of possible values of the ClickHouse `Date` and `DateTime`
+  types. ({issue}`11116`)
+
+## Delta Lake connector
+
+* Improve optimized Parquet writer performance for
+  [structural data types](https://trino.io/docs/current/language/types.html#structural-data-types). ({issue}`13714`)
+* Fix performance regression in reading Parquet files. ({issue}`14094`)
```

Comment:
```suggestion
* Fix performance regression when reading Parquet data. ({issue}`14094`)
```

We may also want to mention which version introduced the regression and add a warning to the corresponding release note page.

Reply:
Regression was introduced in 394. I am not sure if the regression was large enough to be easily noticeable and justify a warning. @kabunchi would you happen to know ?

Reply:
The problem was that the parquet page source generated more pages for largish columns. 
It caused issue only because we had some inefficiency in our connector in the getMetrics implementation and only in one query.
So, not easily noticeable and I don't think it justifies a warning.